hw/chisel/src/test/scala/snax/csr_manager/CsrManagerGenerate.scala

```diff
@@ -4,13 +4,14 @@ import chisel3._
 import org.scalatest.flatspec.AnyFlatSpec
 import chiseltest._
 
-class CsrManagerTopGenerate
-extends AnyFlatSpec {
+class CsrManagerTopGenerate extends AnyFlatSpec {
 
   emitVerilog(
-    new CsrManager(10, 32),
-    Array("--target-dir", "generated/")
-  )    
+    new CsrManager(
+      CsrManagerTestParameters.csrNum,
+      CsrManagerTestParameters.csrAddrWidth
+    ),
+    Array("--target-dir", "generated/csr_manager")
```

Comment:
For these, do we point it into the `hw/chisel/genreated/csr_manager/.` ? Or do we put it at the `../../target/snitch_cluster/generated/.` ?

Reply:
For bug, we put it locally.